src/main/java/frc/robot/subsystems/drivetrain/CommandSwerveDrivetrain.java

```diff
@@ -124,6 +143,27 @@ public CommandSwerveDrivetrain(
         if (Utils.isSimulation()) {
             startSimThread();
         }
+
+        {
+            try {
+                config = RobotConfig.fromGUISettings();
+            } catch (IOException | ParseException e) {
+                throw new RuntimeException(e);
+            }
+            AutoBuilder.configure(
+                    () -> this.getState().Pose,
+                    this::resetPose,
+                    this::getChassisSpeeds,
+                    (ChassisSpeeds speeds) -> this.setControl(AutoReq.withSpeeds(speeds)),
+                    new PPHolonomicDriveController(
+                            new PIDConstants(5.0, 0.0, 0.0), new PIDConstants(5.0, 0.0, 0.0)),
+                    config,
+                    () -> {
+                        var alliance = DriverStation.getAlliance();
+                        return alliance.filter(value -> value == Alliance.Red).isPresent();
+                    },
+                    this);
+        }
```

Comment:
this doesn't need to be in an empty block anymore
```suggestion
        try {
            config = RobotConfig.fromGUISettings();
        } catch (IOException | ParseException e) {
            throw new RuntimeException(e);
        }
        AutoBuilder.configure(
                () -> this.getState().Pose,
                this::resetPose,
                this::getChassisSpeeds,
                (ChassisSpeeds speeds) -> this.setControl(AutoReq.withSpeeds(speeds)),
                new PPHolonomicDriveController(
                        new PIDConstants(5.0, 0.0, 0.0), new PIDConstants(5.0, 0.0, 0.0)),
                config,
                () -> {
                    var alliance = DriverStation.getAlliance();
                    return alliance.filter(value -> value == Alliance.Red).isPresent();
                },
                this);
```